Tic-Tac-Toe.py

```diff
@@ -0,0 +1,171 @@
+#Tic Tac Toe
+
+#create a 3x3 list of blank spaces
```

Comment:
nit: since these comments describe what the function does, they are better suited as doc strings.  e.g.:
```python
def new_board():
        """Create a 3x3 list of blank spaces."""
	board = [[" ", " ", " "], [" ", " ", " "], [" ", " ", " "]]
	return board
```
This makes the purpose of the function discoverable:
```python
print(new_board.__doc__)
# or
help(new_board)
```